### PR TITLE
Hotfix/syal 1276 internal review

### PR DESCRIPTION
**Pull request by HieuLM-AVISPL**

### CODE REVIEW

File: src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java [hunk 1]
```diff
@@ -546,7 +555,7 @@ private void login() {
 	 * @param path url of the request
 	 * @return String full path of the device
 	 */
-	private String buildDeviceFullPath(String path) {
+	public String buildDeviceFullPath(String path) {
```
Comment (trunghieu98) [inline src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java:558]: Please change method to private

Comment (ThanhD-AVISPL) [inline src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java:558]: Please change the method to private. If you using public please explain the reason

File: src/test/java/GudePDU8045CommunicatorTest.java [hunk 2]
```diff
@@ -449,4 +451,25 @@ void testDynamicStatistic() throws Exception {
 		Assertions.assertNotNull(dynamicStats.get("SensorPort02#Temperature(C)"));
 	}
 
+	@Test
```
Comment (ThanhD-AVISPL) [inline src/test/java/GudePDU8045CommunicatorTest.java:454]: Please add javadocs with descriptions and expectation

**Review by ThanhD-AVISPL — APPROVED** [review]

File: src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java (hunk 1 above)
Comment (HieuLM-AVISPL) [reply src/main/java/com/avispl/symphony/dal/avdevices/power/gude/GudePDU8045Communicator.java:558]: I fixed it

File: src/test/java/GudePDU8045CommunicatorTest.java (hunk 2 above)
Comment (HieuLM-AVISPL) [reply src/test/java/GudePDU8045CommunicatorTest.java:454]: I fixed it

**Pull request merged**